### PR TITLE
feat: Switch to 1px outline for all inputs and buttons, as well as use background color in lists

### PR DESCRIPTION
**Pull request by kof**
## Description

Most controls are affected 2px > 1px
Navigator and project settings lists now use background color instead of outline

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: apps/builder/app/builder/features/sidebar-left/sidebar-tabs.tsx [hunk 1]
```diff
@@ -19,17 +20,7 @@ export const SidebarTabs = styled(Tabs, {
...
-    content: '""',
-    position: "absolute",
-    inset: 4,
-    outlineWidth: 2,
-    outlineStyle: "solid",
-    outlineColor: theme.colors.borderFocus,
-    borderRadius: theme.borderRadius[3],
-  },
-};
+const triggerFocusRing = focusRingStyle({ inset: 2 });
```
Comment (TrySound) [inline apps/builder/app/builder/features/sidebar-left/sidebar-tabs.tsx:23]: there is outline-offset property

**Comment by TrySound** [comment]
I barely see outline on my tv
image

**Comment by TrySound** [comment]
I don't see toggle is focused, only tooltip helps
image

**Comment by TrySound** [comment]
Honestly not very accessible change

**Comment by kof** [comment]
> I barely see outline on my tv. Especially near white canvas.

I think the TV screen might be shit. The reference is better screens. I understand that this seems less accessible and that's why we had 2px borders to begin with.

The reality though is that power users who use the platform a lot, get very annoyed by fat outlines, because these become much more invasive perceptionally over time.

We are trading of a first-time user experience for a better power user experience. Honestly a first-time user is unlikely to use keyboard at all, so most beginners won't see the outlines in the first place.

**Comment by kof** [comment]
I didn't come to this conclusion on my own, its the community feedback + I looked at figma and webflow and both are fine with 1px outlines.

File: apps/builder/app/builder/features/sidebar-left/sidebar-tabs.tsx (hunk 1 above)
Comment (kof) [reply apps/builder/app/builder/features/sidebar-left/sidebar-tabs.tsx]: reworked

**Comment by istarkov** [comment]
Login, see corners
image

**Comment by istarkov** [comment]
image
Still 2 px

**Comment by istarkov** [comment]
2px
image

**Comment by kof** [comment]
> image Still 2 px

yes, aware, these are native outlines that are everywhere around the svg icon, we should have used a component, now I leave it for a separate rewrite of these

**Comment by kof** [comment]
> 2px image

intentional

**Comment by istarkov** [comment]
image
Seems like 2px

**Comment by kof** [comment]
> Login, see corners image

seems ok on my machine, weird

image

**Comment by istarkov** [comment]
Link
image

**Comment by istarkov** [comment]
Component
image

**Comment by istarkov** [comment]
image

**Pull request merged**